lib/mayaUsd/ufe/CMakeLists.txt

```diff
@@ -94,6 +94,29 @@ if(CMAKE_UFE_V4_FEATURES_AVAILABLE)
                 UsdShaderNodeDefHandler.cpp
         )
     endif()
+
+    if (${UFE_PREVIEW_VERSION_NUM} GREATER_EQUAL 4008)
+        target_sources(${PROJECT_NAME}
+            PRIVATE
+                UsdShaderAttributeDef.cpp
+        )
+    endif()
+
+
+    if (${UFE_PREVIEW_VERSION_NUM} GREATER_EQUAL 4009)
+        target_sources(${PROJECT_NAME}
+            PRIVATE
+                ProxyShapeCameraHandler.cpp
+                ProxyShapeSceneSegmentHandler.cpp
+        )
+    endif()
+
+    if (${UFE_PREVIEW_VERSION_NUM} GREATER_EQUAL 4010)
```

Comment:
We're in kind of a strange state because we've had a bunch of UFE preview versions go by without getting them into Maya. Everyone is building locally slightly differently. But no QA-testable build with Ufe 0.4.9 is ever going to exist, which means these could all be testing for 0.4.10 (or 0.4.11 if my extra python API changes are merged). That simplifies the cmake. But we don't know which version to check for until something gets merged into Maya, and we can't merge into Maya until this gets merged...

Reply:
It seems to me we can merge this with GREATER_EQUAL 4010 (or 4011) at any time.  Local developer builds will have 0.4.10 (or 0.4.11, as the case may be), so maya-usd will work with the newer version, as well as with the older UFE version.  Am I missing something?